src/services/creation/character/character-creation.service.ts

```diff
@@ -0,0 +1,43 @@
+import _ from "lodash"
+import { Message } from "discord.js"
+import { TextFormats } from "../../../enums/text-formats.enum"
+import { DisplayMessageService } from "../../display-message/display-message.service"
+import { CharacterService } from "../../character/character.service"
+import { MessageFormattingService } from "../../message-formating/message-formatting.service"
+import { StateManagerService } from '../../state-manager/state-manager.service'
+
+export class CharacterCreationService {
+  private static _instance: CharacterCreationService
+
+  public static getInstance(): CharacterCreationService {
+    if(_.isNil(CharacterCreationService._instance)) {
+      CharacterCreationService._instance = new CharacterCreationService()
+    }
+    return CharacterCreationService._instance
+  }
+
+  public init(message: Message) {
+    CharacterService.getInstance().getEntity(message.author.id)
+    .then((characterFound) => {
+      if (!characterFound) {
+        const currentState = StateManagerService.getInstance().getBotState()
+
+        if (currentState.state === 'Normal') {
```

Comment:
Better to use an enum here instead of a string

Reply:
Yes! It will be done in the next pull request.